sktime/forecasting/model_selection/tests/test_split.py

```diff
@@ -325,6 +325,29 @@ def test_sliding_window_splitter_start_with_empty_window(
             )
 
 
+@pytest.mark.parametrize("y", TEST_YS)
+@pytest.mark.parametrize("fh", [*TEST_FHS, *TEST_FHS_TIMEDELTA])
+@pytest.mark.parametrize("step_length", TEST_STEP_LENGTHS)
+def test_expanding_window_splitter_start_with_initial_window_zero(y, fh, step_length):
+    """Test ExpandingWindowSplitter."""
+    initial_window = 0
+    if _inputs_are_supported([fh, step_length, initial_window]):
+        cv = ExpandingWindowSplitter(
+            fh=fh,
+            step_length=step_length,
+            initial_window=initial_window,
+        )
+        train_windows, test_windows, _, n_splits = _check_cv(
+            cv, y, allow_empty_window=True
+        )
+
+        assert np.vstack(test_windows).shape == (n_splits, len(check_fh(fh)))
+    else:
+        match = "Unsupported combination of types"
+        with pytest.raises(TypeError, match=match):
+            ExpandingWindowSplitter(fh=fh, initial_window=0, step_length=step_length)
```

Comment:
```suggestion
            ExpandingWindowSplitter(fh=fh, initial_window=initial_window, step_length=step_length)
```

Reply:
Good catch. Am I doing the right thing by "re-requesting review" each time I complete the changes?

Also, this one auto resolved after I accepted changes, it wasn't me!

Reply:
If you commit suggestion "as is", it is indeed auto resolved, as expected

re-requesting review is not mandatory. at least in my case, I get email notifications on the PR that I participate in